llvm/lib/CodeGen/AsmPrinter/DwarfUnit.cpp

```diff
@@ -1762,16 +1762,26 @@ void DwarfUnit::emitCommonHeader(bool UseOffsets, dwarf::UnitType UT) {
     Asm->emitInt8(Asm->MAI->getCodePointerSize());
   }
 
-  // We share one abbreviations table across all units so it's always at the
-  // start of the section. Use a relocatable offset where needed to ensure
-  // linking doesn't invalidate that offset.
+  // With the possibility of multiple compile units, every DwarfUnit has its own
```

Comment:
` With the possibility of multiple compile units` what does this mean?

Reply:
What this means is that, there is a possibility that we can have multiple compile units in one translation unit. I have reworded the comment to make sure it is more clear, please let me know if that is okay?